app/Exceptions/ModuleInstallationError.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Exceptions;
+
+class ModuleInstallationError extends AbstractHttpException
+{
+    public function __construct()
```

Comment:
Pass in the name of the module here, so it'll get logged

Reply:
Hmm okay..